app/views/proprietor/accounts/edit.html.erb

```diff
@@ -34,7 +34,7 @@
           </div>
 
           <h3><%= t(".account_settings") %></h3>
-          <% current_account.live_settings.each do |key, value| %>
+          <% @account.live_settings.each do |key, value| %>
```

Comment:
Should this change be parked until question in FIXME in app/views/shared/_settings.html.erb is resolved?  I am not sure if the tests will continue to pass if this is changed back to `current_account` so if that is the case then please ignore this comment.

Reply:
This change is okay to stay. `current_account` was incorrect here too and changing it to `@account` doesn't break anything. 

`current_account` was essentially returning `Account.new`, while `@account` is the actual Account the form is editing

Reply:
Well, then that should absolutely stay.  Thanks for the clarification!